packages/block-editor/src/components/colors-gradients/control.js

```diff
@@ -181,6 +181,9 @@ function ColorGradientControlSelect( props ) {
 	colorGradientSettings.disableCustomColors = ! useEditorFeature(
 		'color.custom'
 	);
+	colorGradientSettings.disableCustomGradients = ! useEditorFeature(
+		'gradients.custom'
```

Comment:
I think this should be `gradient.custom`. Fix at https://github.com/WordPress/gutenberg/pull/25040